CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+
+3. Open up a new browser or tab and go to the qc-coders repo: 
+    [Q-C Coders](https://github.com/QC-Coders/qc-coders.github.io)
+
+4. Click on the "Fork" button
+    -   You now have the repo forked under your own GitHub account.
+
+5. After the repository has been forked, you will be taken to your copy of
+     the Q-C Coders repo at `your_Username_Here/qc-coders.github.io`
+    -  Make sure to replace `your_Username_Here` with your GitHub username
+
+
+### Cloning
+
+*What are these cloning instructions doing?*
+
+Cloning is simply putting the code that you previously forked (copied to your own GitHub repo) onto your local machine. 
+
+
+1. Open Git Bash Shell and navigate to the location you're going to save this project locally.
+    -   Make a note of where this location is as you will use it later.
+
+2. Type or copy: `git clone https://github.com/your_Username_Here/qc-coders.github.io.git`
+	-   This will download the entire qc-coders repo to your projects directory.
+
+
+### Configuring the Upstream
+
+*What are these upstream instructions doing?*
+
+Upstream is a way of keeping the files on your computer updated with any changes that take place in production. For instance, while you are making changes, someone else make a pull-request that gets merged...Whoa? Wait! Pull-request? Merge? Let's explain these terms. A pull-request is simply someone requesting to push updates, fixes or changes to the production server. A merge takes place when one of us accept the those changes. So, now your computer doesn't have the latest changes, so this is just what upstream does. Every time you use the upstream command (explained later), Git will check to make sure your files are the same as production.
+
+
+1. Using Git Bash navigate to the directory where you previously cloned the new qc-coders directory on your machine.
+    -   Remember when this document said "Make a note of where this location is"? Yep, that's the location!
+
+2. Add the upstream to the qc-coders production repo:
+	`git remote add upstream https://github.com/QC-Coders/qc-coders.github.io.git`
+
+    -   Great! You are almost ready to contrubute to qc-coders!
+
+
+### Maintaining your Fork
+
+*Didn't I already do this?*<br/>
+
+Remember, a long time ago when you forked the production repo and then cloned it. (Essentially, putting qc-coders into your own GitHub and then making a copy of the code on your computer). Then we setup that upstream so you can get the latest and greatest changes. Well, this is what you will need to do **every** time you wish to keep that code updated. Oh, you thought it was going to do this automatically? Nope! (Guess, you have another project to add to your list to make this happen!) The term used for all of this is called: rebasing. Yes, another terrible name.
+
+So each time you want to update your code base or (*most importantly*) before making a pull-request, follow these steps below:
+
+
+1. Make sure you are on the `staging branch`
```

Comment:
This seems like a jump; should the instructions for creating a staging branch come first?